src/Bitmap/BitmapFile.h

```diff
@@ -22,14 +22,15 @@ class BitmapFile
 	std::vector<Color> palette;
 	std::vector<uint8_t> pixels;
 
+	// Uses a default 4 byte pitch
 	static BitmapFile CreateDefaultIndexed(uint16_t bitCount, uint32_t width, uint32_t height);
 
-	// BMP Reader only supports Indexed Color palettes (1, 2, and 8 bit BMPs).
+	// BMP Reader only supports indexed color palettes (1, 2, and 8 bit BMPs).
 	static BitmapFile ReadIndexed(const std::string& filename);
 	static BitmapFile ReadIndexed(Stream::BidirectionalSeekableReader& seekableReader);
 
-	// BMP Writer only supporting Indexed Color palettes (1, 2, and 8 bit BMPs).
-	// @indexedPixels: Must include padding to fill each image row out to the next 4 byte memory border (pitch).
+	// BMP Writer only supports indexed color palettes (1, 2, and 8 bit BMPs).
+	// @indexedPixels: Must include padding to fill each image row out to the next byte memory border (pitch). 4 byte pitch is typical.
```

Comment:
This comment sounds a bit confusing. I think it's made more confusing because the parameters don't allow you to specify the pitch. I assume the pitch comes from the `BmpHeader` struct stored as a member variable of the class.

Edit: Ok, I see the `pitch` parameter was added later. The comment is still a little confusing though. Maybe phrase as:
"@indexedPixels: Size must be padded to a multiple of pitch. (4 byte pitch is typical)"

It would also help to be consistent about specifying pitch as either bytes or bits. I thought I saw somewhere else use 32-bits, whereas here it's using 4 bytes.

Reply:
I'll update the comment as suggested. I'll try to standardize on using the term 4 bytes for standard. The variable takes bytes, so it makes more sense to reference bytes in the comments to not confuse the issue.

-Brett